_pages/contact.md

```diff
@@ -6,11 +6,16 @@ order: 6
 ---
 ## Mailing List
 There is a Google Groups mailing list for discussions. To be added to the mailing list for this project, please do the following:<br />
-1) Ask to join the group at [this link](https://groups.google.com/group/pyhc-list/subscribe)<br />
-2) You should receive a follow up email from doing Step 1. In that email, click 'Join This Group' to confirm your subscription. It should then send you back to the Google Groups page, with a message saying your email has been added to the groups.<br />
+1) Ask to join the group at [this link](https://groups.google.com/group/pyhc-list/subscribe).* <br />
```

Comment:
I suggest that this should not be invite only but an open mailing list that anyone can join without gatekeepers.

Reply:
It is - the setting was just wrong at first

Reply:
What alex said